src/test/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/caseworker/ManageDocumentsCallbackHandlerTest.java

```diff
@@ -96,25 +111,102 @@ void shouldNotReturnErrorsIfThereAreDifferences() {
             Assertions.assertNull(response.getErrors());
         }
 
-        @Test
-        void shouldReturnErrorsIfThereAreNoDifferences() {
-            CaseDetails caseDetails = CaseDetails.builder().build();
+        @Nested
+        @DisplayName("No differences test")
+        class NoDifferencesTest {
+            private CaseDetails caseDetails;
+            private CaseDetails caseDetailsBefore;
+            private CallbackParams callbackParams;
 
-            CallbackRequest request = CallbackRequest.builder()
-                .caseDetails(caseDetails)
-                .caseDetailsBefore(caseDetails)
-                .build();
+            @BeforeEach
+            void setUp() {
 
-            CallbackParams callbackParams = CallbackParams.builder()
-                .type(CallbackType.MID)
-                .request(request)
-                .build();
+                caseDetails = CaseDetails.builder().build();
+                caseDetailsBefore = CaseDetails.builder()
+                    .data(ImmutableMap.of("different", "data"))
+                    .build();
 
-            AboutToStartOrSubmitCallbackResponse response
-                = (AboutToStartOrSubmitCallbackResponse) handler.handle(callbackParams);
+                CallbackRequest request = CallbackRequest.builder()
+                    .caseDetails(caseDetails)
+                    .caseDetailsBefore(caseDetailsBefore)
+                    .build();
 
-            Assertions.assertNotNull(response.getErrors());
-            assertThat(response.getErrors(), contains(ManageDocumentsCallbackHandler.NO_CHANGES_ERROR_MESSAGE));
+                callbackParams = CallbackParams.builder()
+                    .type(CallbackType.MID)
+                    .request(request)
+                    .build();
+
+            }
+
+            @Test
+            void shouldReturnErrorsIfThereAreNoDifferencesNullStaffDocuments() {
+
+                CCDCase ccdCase = CCDCase.builder().build();
+
+                when(caseDetailsConverter.extractCCDCase(eq(caseDetails))).thenReturn(ccdCase);
```

Comment:
Would be nice to see mocking of caseDetailsBefore. I guess the assumption is that the value is null.

Reply:
`caseDetailsBefore` mock is missing. I have now updated.